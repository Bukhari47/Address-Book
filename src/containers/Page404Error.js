import React from "react";

function Page404Error() {
  return (
    <div style={{ textAlign: "center" }}>Opps...!!! Page not found...!!!</div>
  );
}

export default Page404Error;
