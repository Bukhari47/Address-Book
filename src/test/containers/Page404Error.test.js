import React from "react";
import Page404Error from "../../containers/Page404Error";
import renderer from "react-test-renderer";
import matchMedia from "../../matchMedia.mock";

it("should display 404 error", () => {
  const tree = renderer.create(<Page404Error />);
  expect(tree).toMatchSnapshot();
});
