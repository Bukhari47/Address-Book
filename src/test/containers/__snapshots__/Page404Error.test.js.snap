// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display 404 error 1`] = `
<div
  className="ant-row"
  style={Object {}}
>
  <div
    className="ant-col"
    style={Object {}}
  >
    <span
      className="ant-typography"
      style={
        Object {
          "WebkitLineClamp": undefined,
        }
      }
    >
      <strong>
        Opps...!!! Page not found...!!!
      </strong>
    </span>
  </div>
</div>
`;
